docs/source/contributing/developing.adoc

```diff
@@ -316,6 +316,13 @@ The following options are available:
 
 Tags::
 Use `tags` to ensure that scenarios and features containing at least one of the selected tags are executed.
+To select particular feature you can use its name as a tag.
+For example, `cmd-config.feature` contains and can be called via `cmd-config` tag.
```

Comment:
I think there are few words missing here `For example, 'cmd-config.feature' contains <something missing like @cmd-config tag> and`

Reply:
I agree, and I still find this wording confusing.